content/en/docs/examples/virtual-machines/single-network/index.md

```diff
@@ -60,7 +60,7 @@ following commands on a machine with cluster admin privileges:
 
     {{< text bash >}}
     $ kubectl create namespace istio-system
-    $ kubectl create secret generic cacerts -n istio-system \
+    $ kubectl create secret generic istio-ca-cert -n istio-system \
```

Comment:
Is this a design change that also needs more doc updates, for example here: https://preliminary.istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/#plugging-in-existing-certificates-and-key?

Reply:
This is hard coded in https://github.com/istio/istio/blob/master/security/tools/generate_cert/main.go#L98 as istio-ca-secret.  I don't know if the tool is wrong or the docs are wrong, but they should agree on the name.